drivers/i2c/i2c_nrfx_twim_common.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
  * no longer ISR safe. This macro let's us check if we will be requesting/releasing
  * power domains and determines PM device ISR safety value.
  */
-#define I2C_PM_ISR_SAFE(idx)									\
```

Comment:
are those white space changes intentional? I see that  lines 46-47  exceed 100 chars. Maybe those shall be fixed?